engines/director/lingo/lingo-builtins.cpp

```diff
@@ -1823,9 +1823,19 @@ void LB::b_numberoflines(int nargs) {
 void LB::b_numberofwords(int nargs) {
 	Datum d = g_lingo->pop();
 
-	warning("STUB: b_numberofwords");
-	d.toInt();
-	d.u.i = 0;
+	d.toString();
+	int numberofwords = 0;
+	Common::String contents = *d.u.s;
+	for (uint32 i=1; i < d.u.s->size(); i++) {
```

Comment:
Formatting is incorrect `i = 1`

Reply:
Is there a formatter that can be automatically used for scummvm?

Reply:
Yes, mentioned right on the bottom of the Code Formatting Guidelines: https://wiki.scummvm.org/index.php?title=Code_Formatting_Conventions

I highly suggest exploring our Developer Central.